src/app/component/MapView/MapView.js

```diff
@@ -12,7 +12,7 @@ function MapView(props) {
   });
   const position = [state.lat, state.lng];
   return (
-    <Map center={position} zoom={state.zoom} style={MapStyle} data-testid="map">
+    <Map center={position} zoom={state.zoom} style={MapStyle} id="leaflet-map">
```

Comment:
This could be problematic if we want to have multiple MapViews on one page.

Reply:
Maybe use a bit more unique id: leaflet-missions-map or something like that. I suck at naming.

Reply:
**Update: React.Fragment only supports {key} and {children}, so defaulting back to `div`**

Good point, `react-leaflet` doesn't support `data-testid` unfortunately. Since this is a selector for tests, another solution would be to wrap everything with a `<div>` so we can use `getByTestId` in the tests.
```
<div data-testid="leaflet-mapview">
  <Map ... />
</div>
```

Reply:
Hmm would leaflet work with a class name?